rules/README.md

```diff
@@ -0,0 +1,67 @@
+# Re
+
+Re service provides a barebones HTTP API for development of a Mainflux
+service.
+
+## Configuration
+
+The service is configured using the environment variables presented in the
+following table. Note that any unset variables will be replaced with their
+default values.
+
+| Variable              | Description                                             | Default |
+|-----------------------|---------------------------------------------------------|---------|
+| MF_RE_LOG_LEVEL   | Log level for re service (debug, info, warn, error) | error   |
+| MF_RE_HTTP_PORT   | Re service HTTP port                                | 9021    |
+| MF_RE_SERVER_CERT | Path to server certificate in pem format                |         |
+| MF_RE_SERVER_KEY  | Path to server key in pem format                        |         |
+| MF_JAEGER_URL         | Jaeger server URL                                       |         |
+| MF_RE_SECRET      | Re service secret                                   | secret  |
+
+## Deployment
+
+The service itself is distributed as Docker container. The following snippet
+provides a compose file template that can be used to deploy the service container
+locally:
+
+```yaml
+version: "3"
+services:
+  re:
+    image: mainflux/re:[version]
+    container_name: [instance name]
+    ports:
+      - [host machine port]:[configured HTTP port]
+    environment:
+      MF_RE_LOG_LEVEL: [Kit log level]
+      MF_RE_HTTP_PORT: [Service HTTP port]
+      MF_RE_SERVER_CERT: [String path to server cert in pem format]
+      MF_RE_SERVER_KEY: [String path to server key in pem format]
+      MF_JAEGER_URL: [Jaeger server URL]      
+      MF_RE_SECRET: [Re service secret]
+```
```

Comment:
As @mteodor proposed use a link of the docker-compose here

Reply:
check the other readme files
https://github.com/mainflux/mainflux/tree/master/things
```
## Deployment

The service itself is distributed as Docker container. Check the [`things `](https://github.com/mainflux/mainflux/blob/master/docker/docker-compose.yml#L167-L194) service section in 
docker-compose to see how service is deployed.
```